### DISCUSSION TITLE
ZbBind door sensor with light switch

### DISCUSSION THREAD

**Discussion by marjankow** [arendst/Tasmota#21972] [Q&A] [answered]
Hello is it posible to zbbind door sensor
{"ZbReceived":{"tarrace_entrance":{"Device":"0x569F","Name":"tarrace_entrance","0500?00":"010000010000","ZoneStatusChange":1,"ZoneStatusChangeZone":1,"Contact":1,"Endpoint":1,"LinkQuality":100}}}

{"ZbReceived":{"tarrace_entrance":{"Device":"0x569F","Name":"tarrace_entrance","0500?00":"000000010000","ZoneStatusChange":0,"ZoneStatusChangeZone":1,"Contact":0,"Endpoint":1,"LinkQuality":102}}}

with light switch 
{"ZbReceived":{"out_light":{"Device":"0x92C8","Name":"out_light","Power":1,"Endpoint":1,"LinkQuality":94}}}
Following not working:
ZbBind {"Device":"tarrace_entrance","ToDevice":"out_light","Endpoint":1,"ToEndpoint":1,"Cluster":6}
Is it possible to convert Contact on Power somehow ?

**Comment by s-hadinger** [comment]
Unfortunately not. The first device sends a status notification from cluster 0x0500 which the light switch will not understand. 

You can do Tasmota rules for this

**Reply by marjankow** [reply]
Ok thanks for fast reply. So it's not possible to have target cluster in ZbBind command?

**Reply by s-hadinger** [reply, marked as answer]
No. This is part of zigbee spec.
